src/pages/confirmation/confirmation.styles.js

```diff
@@ -5,8 +5,8 @@ export const useStyles = makeStyles((theme) => ({
   confirmation: {
     background: `url(${BG}) no-repeat `,
     backgroundSize: 'cover',
```

Comment:
write background size and position in background property

Reply:
fixed